docs/windows.md

```diff
@@ -1,142 +1,94 @@
-command away.
-
-So use the next steps below that start with "Manual Way" if that's the
-way you want, otherwise scroll down a bit and use the "Cygwin Way"
-steps below.
+## Cygwin
+Install Cygwin (https://www.cygwin.com)  
+During the install, select the devel packages `git` & `make`, archive package `unzip`, and net package `curl`  
+Run the following inside a normal `cmd` prompt (Not the Cygwin prompt, we only need Cygwin's tools)  
+An explanation is below.
+```
+SET PATH=%PATH%;\cygwin64\bin
+mkdir %GOPATH%\src\github.com\ipfs
+cd %GOPATH%\src\github.com\ipfs
+git clone https://github.com/ipfs/go-ipfs.git
+cd %GOPATH%\src\github.com\ipfs\go-ipfs
+make install
+```
+|      |      |
+| ---: | :--- |
+|`SET PATH=%PATH%;\cygwin64\bin`           |Add Cygwin's bin to PATH; Defaults to: (\cygwin64\bin)|
+|`mkdir %GOPATH%\src\github.com\ipfs`<br/>`cd %GOPATH%\src\github.com\ipfs`<br/>`git clone https://github.com/ipfs/go-ipfs.git`       |Fetch / Update `go-ipfs` source|
+|`cd %GOPATH%\src\github.com\ipfs\go-ipfs` |Change to `go-ipfs` directory|
+|`make install`                            |Build and install to `%GOPATH%\bin\ipfs.exe`|
 
-## Manual Way: download and install dependencies
+You can permanently append the Cygwin bin to your `PATH` so that on subsequent builds you can simply run
+```
+cd %GOPATH%\src\github.com\ipfs\go-ipfs
+make install
+```
 
-The following commands should download or update go-ipfs dependencies
-and then install them:
+## Minimal
+While it's possible to build `go-ipfs` with `go` alone, we'll be using `git` and `gx` for practical source management.  
+You can use whichever version of `git` you wish but the Windows builds at <https://git-scm.com> are recommended. `git` must be in your `PATH` for `go get` to recognize and use it.  
+`gx` and its go-hook `gx-go` will be built by us. Alternativley you may use [prebuilt binaries](https://dist.ipfs.io/#gx) by placing them in your `PATH` and skipping `go get` for `gx` and `gx-go` below.  
 
 ```
+SET PATH=%PATH%;%GOPATH%\bin
 go get -u github.com/whyrusleeping/gx
 go get -u github.com/whyrusleeping/gx-go
+go get -u -d github.com/ipfs/go-ipfs
 cd %GOPATH%/src/github.com/ipfs/go-ipfs
 gx --verbose install --global
-```
-
-## Manual Way: build go-ipfs
-
-To actually build go-ipfs, first go to the cmd/ipfs directory:
-
-```
 cd cmd\ipfs
 ```
-
-Then get the current Git commit:
-
-```
-git rev-parse --short HEAD
-```
-
-It will output a small number of hex characters that you must pass to
-the actual build command (replace XXXXXXX with these characters):
-
-```
-go install -ldflags="-X "github.com/ipfs/go-ipfs/repo/config".CurrentCommit=XXXXXXX"
-```
-
-After that ipfs should have been built and should be available in
-"%GOPATH%\bin".
-
-You can check that the ipfs you built has the right version using:
-
-```
-ipfs version --commit
-```
-
-It should output something like "ipfs version 0.4.0-dev-XXXXXXX" where
-XXXXXXX is the current commit that you passed to the build command.
```

Comment:
I think you should keep the step of verifying the install here. (You could remove the “where		
 -XXXXXXX is the current commit that you passed to the build command” bit since you’ve abstracted that away with a command + env var.)

It might also be useful to give some basic advice on what to do if they *don’t* get the expected output, even if it’s just “post to https://discuss.ipfs.io .”